.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: General PowerShell Question
+    url: https://github.com/PowerShell/PowerShell/issues/
+    about: PowerShell module or language related question
+  - name: Microsoft Graph Module Question
+    url: https://github.com/microsoftgraph/msgraph-sdk-powershell/issues/
+    about: Question about Microsoft Graph PowerShell Module 
```

How about linking to autorest.powershell repo?
```suggestion
    about: Question about Microsoft Graph PowerShell Module
  - name: AutoRest PowerShell Generator Question
    url: https://github.com/Azure/autorest.powershell/issues
    about: Questions about the PowerShell plugin of AutoRest
```

---

Added